src/components/customer-resource-coverage-fields/customer-resource-coverage-fields.js

```diff
@@ -0,0 +1,243 @@
+import React, { Component } from 'react';
+import { Field, FieldArray } from 'redux-form';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import Datepicker from '@folio/stripes-components/lib/Datepicker';
+import Button from '@folio/stripes-components/lib/Button';
+import IconButton from '@folio/stripes-components/lib/IconButton';
+
+import styles from './customer-resource-coverage-fields.css';
+import { formatISODateWithoutTime } from '../utilities';
+
+export default class CustomerResourceCoverageFields extends Component {
+  static propTypes = {
+    packageCoverage: PropTypes.object, // eslint-disable-line react/no-unused-prop-types
```

Comment:
minor nit, but why not just `eslint-disable` this rule for the block instead of line-by-line?

Reply:
No particular reason 😎 Actually, after looking at it again, I decided to leave it that way, so if we add any props in the future they're immediately checked by ESLint. I'd probably forget to take future props out of the block ignore.